src/libraries/System.Text.Json/tests/Serialization/PropertyVisibilityTests.NonPublicAccessors.cs

```diff
@@ -350,5 +353,23 @@ private class ClassWithProtectedField_WithJsonIncludeProperty
             [JsonInclude]
             protected string MyString = null;
         }
+
+        private class ClassWithPrivate_InitOnlyProperty_WithJsonIncludeProperty
+        {
+            [JsonInclude]
+            private string MyString { get; init; }
```

Comment:
Is there a way to (de)serialize non-public properties/fields? I was somewhat confused what `JsonInclude` was enabling in these scenarios until I realized that only bypasses get/set accessibility and no property/field accessibility.

Reply:
Not at the moment. There's an [issue](https://github.com/dotnet/runtime/issues/31511) to allow this in the future. We can do this following the [codegen effort](https://github.com/dotnet/runtime/issues/1568).